demo/demo.html

```diff
@@ -0,0 +1,100 @@
+<!doctype html>
+<html>
+
+<head>
+  <title>Block Types Demo</title>
+
+  <script src="../node_modules/react/dist/react.js"></script>
+  <script src="../node_modules/react-dom/dist/react-dom.js"></script>
+  <script src="../element/build/index.js"></script>
+  <script src="../blocks/build/index.js"></script>
+
+  <script src="https://fiddle.azurewebsites.net/tinymce/nightly/tinymce.min.js"></script>
+
+  <link rel="Stylesheet" href="../editor/build/style.css">
+
+  <script src="../editor/build/index.js"></script>
+
+  <script>
+    const setupBlock = (slug, attributes) => {
+      let row = document.querySelector('tr.' + slug);
+      let settings = wp.blocks.getBlockSettings('core/' + slug);
+      let comp = settings.edit(attributes, () => { });
+      ReactDOM.render(comp, row.querySelector('.editor'));
+      let saved = settings.save(attributes);
+      ReactDOM.render(saved, row.querySelector('.saved'));
+    };
+
+    const setupList = () => {
+      // let slug = wp.blocks.
+    };
+
+    var go = function () {
+      wp.editor.createEditorInstance( 'editor', { content: 'I am here', addBlock: function () {
+        debugger;
+      }});
+      setupBlock('text', { value: 'This is text' })
+      setupBlock('heading', { nodeName: 'h2', value: 'This is a heading' });
+      setupBlock('list', {
+        nodeName: 'ol',
+        items: [
+          { value: 'hi' },
+          { value: 'there' }
+        ]
+      })
+    };
+  </script>
+
+  <style>
+    table {
+      width: 100%;
+      border: 1px solid black;
+      border-collapse: collapse;
+    }
+
+    table td {
+      width: 50%;
+      border: 1px solid black;
+    }
+  </style>
+</head>
+
+<body onload="go();">
+  <div id="editor"></div>
+  <table>
+    <thead>
+      <tr>
+        <th>Editable View</th>
+        <th>Saved View</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr class="text">
+        <td>
+          <div class="editor" />
+        </td>
+        <td>
+          <div class="saved" />
+        </td>
+      </tr>
+      <tr class="heading">
+        <td>
+          <div class="editor" />
+        </td>
+        <td>
+          <div class="saved" />
+        </td>
+      </tr>
+      <tr class="list">
+        <td>
+          <div class="editor" />
+        </td>
+        <td>
+          <div class="saved" />
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</body>
+
+</html>
```

Comment:
I think this demo page should be removed, we already have the default post content viewable in the plugin page.